pkg/agent/handler/accessrequest.go

```diff
@@ -186,10 +186,17 @@ func (h *accessRequestHandler) onError(_ context.Context, ar *management.AccessR
 
 // onDeleting deprovisions an access request and removes the finalizer
 func (h *accessRequestHandler) onDeleting(ctx context.Context, ar *management.AccessRequest) {
+	log := getLoggerFromContext(ctx)
+
+	app, err := h.getManagedApp(ctx, ar)
+	if err != nil {
+		log.WithError(err).Error("error getting managed app")
+		h.onError(ctx, ar, err)
```

Comment:
@mynecker Could we return the err here if one is hit? All else looks good, so after this change it will be merged and a release made soon.

Reply:
@jcollins-axway 

I can just 'return' if we hit an error, is that sufficient? 

```
	app, err := h.getManagedApp(ctx, ar)
	if err != nil {
		log.WithError(err).Error("error getting managed app")
		h.onError(ctx, ar, err)
		return
	}
```

Reply:
yes, sorry, you are correct that this method does not return a type. The onError will update the resource with the error

Reply:
@mynecker if you can make this change we can get this merged ahead of the next release of the sdk

Reply:
@jcollins-axway should handle errors now as expected

Reply:
Thanks!